plugins/spacing.js

```diff
@@ -30,20 +19,24 @@ export const messages = ruleMessages(ruleName, {
   },
 })
 
-const meta = {
-  fixable: true,
+// Props that we want to check
+const propList = ['padding', 'margin', 'top', 'right', 'bottom', 'left']
+// Values that we want to ignore
+const valueList = ['${']
+
+const sizes = primitivesVariables('spacing')
+
+// Add +-1px to each value
```

Comment:
Is this change to support cases where people decrease spacing to accommodate border width?

Reply:
The idea in the spacing thing was if someone wrote `padding: 7px;` we'd fail but also suggest they use `--base-size-8` since it's 1 px off from the preferred value